README.md

```diff
@@ -1,28 +1,42 @@
 # cccatalog-frontend
 
-> Creative Commons Search
+## About
 
-## Build Setup
+Repository containing the [CC Search](https://ccsearch.creativecommons.org/) frontend code.
+
+CC Search aims to private the front door to a comprehensive commons of CC and public domain materials, in order to maximize not only discovery, but use and engagement with all of that content.
```

Comment:
I think "private" should be "provide"

Reply:
You may want to rephrase this to be simpler because this sounds a bit too marketing-y for a code repo. Maybe something like "CC Search is an interface to search for content that is licensed under Creative Commons licenses or in the public domain. "